packages/api/src/router/testSupport/userPreparation/clerkUsers.ts

```diff
@@ -10,10 +10,9 @@ export const deleteOldTestUser = async () => {
     limit: 500,
   });
 
-  const NUMBERS_USER = /\d{5,10}.*@/; // jim+010203@thenational.academy
   const testUsers = result.data.filter((u) => {
     const email = u.primaryEmailAddress?.emailAddress ?? "";
-    return email.startsWith("test+") || email.match(NUMBERS_USER);
+    return email.startsWith("test+");
```

Comment:
The numbers-in-email check is removed here, but logic is part of `hasTestCharacters`?

Reply:
Yeah, I think that the e2e test account should stick to their own logic of having a user pool with a finite length. They will cycle a lot quicker than the 1 month on manually created test accounts (hasTestCharacters).